samples/LargeResponseApp/LargeResponseApp.csproj

```diff
@@ -11,4 +11,8 @@
     <ProjectReference Include="..\..\src\Kestrel\Kestrel.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Compilers" Version="$(MicrosoftNETCoreCompilersVersion)" PrivateAssets="All" />
```

Comment:
This is required because when we build entire repo in one invocation having two versions of compilers in a single process causes load exception.